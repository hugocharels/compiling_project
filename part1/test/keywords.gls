LET
BE
END
IF
THEN
ELSE
WHILE
REPEAT
OUT
IN