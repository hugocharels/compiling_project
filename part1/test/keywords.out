token: LET             lexical unit: LET
token: BE              lexical unit: BE
token: END             lexical unit: END
token: IF              lexical unit: IF
token: THEN            lexical unit: THEN
token: ELSE            lexical unit: ELSE
token: WHILE           lexical unit: WHILE
token: REPEAT          lexical unit: REPEAT
token: OUT             lexical unit: OUTPUT
token: IN              lexical unit: INPUT